### PR TITLE
Add guest agent file system list to VM disks tab

### PR DESCRIPTION
**Pull request by pcbailey**

### CODE REVIEW

File: frontend/packages/kubevirt-plugin/src/components/vm-disks/guest-agent-file-systems.tsx [hunk 1]
```diff
@@ -0,0 +1,130 @@
...
+        name: fs.getDiskName(),
+        fileSystemType: fs.getType(),
+        mountPoint: fs.getMountPoint(),
+        totalBytes: humanizeBinaryBytes(fs.getTotalBytes()).string,
+        usedBytes: humanizeBinaryBytes(fs.getUsedBytes()).string,
+      },
+    }));
+
+  const body = () => {
+    if (vmStatusBundle.status !== VMStatus.RUNNING) {
```
Comment (yaacov) [inline frontend/packages/kubevirt-plugin/src/components/vm-disks/guest-agent-file-systems.tsx:96]: can we replace this with the helper `getGuestAgentFieldNotAvailMsg` ?

File: frontend/packages/kubevirt-plugin/src/components/vm-disks/vm-disks.tsx [hunk 2]
```diff
@@ -192,9 +204,32 @@ export const VMDisksFirehose: React.FC<VMLikeEntityTabProps> = ({ obj: vmLikeEnt
     }),
   ];
 
+  const vm =
+    kindObj === VirtualMachineModel && isVM(vmLikeEntity)
```
Comment (yaacov) [inline frontend/packages/kubevirt-plugin/src/components/vm-disks/vm-disks.tsx:208]: I see that this is how we do it in other places too ... maybe in a follow up - IMHO we can make a helper method to extract vm and vmi from the props, and use in all the places we have this code ... ?

File: frontend/packages/kubevirt-plugin/src/k8s/wrapper/vm/guest-agent-info/guest-agent-fs-info-wrapper.ts [hunk 3]
```diff
@@ -14,5 +14,5 @@ export class GuestAgentFSInfoWrapper extends Wrapper<
   }
 
   getDisks = (): GuestAgentFileSystemWrapper[] =>
-    this.data?.disks.map((disk) => new GuestAgentFileSystemWrapper(disk));
+    this.data?.disks?.map((disk) => new GuestAgentFileSystemWrapper(disk));
```
Comment (yaacov) [inline frontend/packages/kubevirt-plugin/src/k8s/wrapper/vm/guest-agent-info/guest-agent-fs-info-wrapper.ts:17]: nice +1

File: frontend/packages/kubevirt-plugin/src/components/vm-disks/guest-agent-file-systems.tsx (hunk 1 above)
Comment (pcbailey) [reply frontend/packages/kubevirt-plugin/src/components/vm-disks/guest-agent-file-systems.tsx]: Done!

File: frontend/packages/kubevirt-plugin/src/components/vm-disks/vm-disks.tsx (hunk 2 above)
Comment (pcbailey) [reply frontend/packages/kubevirt-plugin/src/components/vm-disks/vm-disks.tsx:212]: Agreed!

**Pull request merged**